lib/oscap_tests.pm

```diff
@@ -123,186 +711,541 @@ sub upload_logs_reports {
     };
 =cut
 
-sub oscap_security_guide_setup {
+sub get_tests_config {
+    # Get the tests configuration file from repository
+    my $config_file_name = "openqa_config.conf";
+    my $url = "https://gitlab.suse.de/seccert-public/compliance-as-code-compiled/-/raw/main/content/";
```

Comment:
we are revealing the information of this file somehow all over the places in comments in this PR so I guess the file we are fetching here is not under some security concern even if it is on that repo, I'm wondering if we could simplify it and just have this file stored in this repo under data/ instead of fetching it. It might get out of sync, but doesn't look like it has much information... wdyt?

Note: we usually use [get_test_data](https://open.qa/api/testapi/#_get_test_data) for fetching from data/ folder.

Reply:
It is complicated because:
1. Improvement work constantly going on by Hardening team for multiple profiles and results/profiles  are constantly changing.
2. Every 3 months there are OSCAP release and results will change.
3. There are need to test profiles from different sources.
For this reason I made gitlab repository where OSCAP under test, expected results, configuration and exclusions are stored.
So we can do changes there quickly without PRs.
When SCAP profiles will stabilize - we will consider to move config part to openQA.